### PR TITLE
Update 6.0.3xx with the 6.0.200 version of files

### PR DESCRIPTION
**Pull request by marcpopMSFT**

### CODE REVIEW

**Comment by eerhardt** [comment]
Is this PR going forward? It looks like bugs that were fixed in dotnet/linker in 6.0.2xx SDK aren't in the latest 6.0.3xx SDK versions.

cc @jonathanpeppers @sbomer @vitek-karas

File: src/Tasks/Common/Resources/xlf/Strings.it.xlf [hunk 1]
```diff
@@ -224,7 +224,7 @@
       </trans-unit>
       <trans-unit id="CompressionInSingleFileRequiresSelfContained">
         <source>NETSDK1176: Compression in a single file bundle is only supported when publishing a self-contained application.</source>
-        <target state="translated">NETSDK1176: la compressione in un unico bundle di file è supportata solo quando si esegue la pubblicazione di un'applicazione indipendente.</target>
+        <target state="new">NETSDK1176: Compression in a single file bundle is only supported when publishing a self-contained application.</target>
```
Comment (baronfel) [inline src/Tasks/Common/Resources/xlf/Strings.it.xlf:227]: I think we need to revert all of these translations changes before merge

**Review by baronfel — DISMISSED** [review]

File: src/Tasks/Common/Resources/xlf/Strings.it.xlf (hunk 1 above)
Comment (baronfel) [reply src/Tasks/Common/Resources/xlf/Strings.it.xlf]: reverted those changes.

**Comment by baronfel** [comment]
@joeloff has concerns about the darc-int packages too that we'll need to talk about tomorrow.

**Review by v-wuzhai — APPROVED** [review]

**Pull request merged**